manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FunctionsTest.scala

```diff
@@ -367,9 +367,9 @@ In case all arguments are +NULL+, +NULL+ will be returned.""",
                        "search" -> "An expression that returns a string to search for",
                        "replace" -> "An expression that returns the string to replace the search string with"),
       text = "`replace()` returns a string with the search string replaced by the replace string. It replaces all occurrences.",
-      queryText = "return replace(\"hello\", \"l\", \"w\")",
+      queryText = "RETURN replace(\'hello\', \'l\', \'w\')",
```

Comment:
When changed to single quotes I guess the escapes are no longer necessary, but they don't seem to have any ill effect either.